icu4c/source/common/BUILD

```diff
@@ -92,6 +92,170 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "errorcode",
+    srcs = [
+        "errorcode.cpp",
+    ],
+    includes = ["."],
+    deps = [
+        ":headers",
+    ],
+    local_defines = [
+        "U_COMMON_IMPLEMENTATION",
+    ],
+)
+
+cc_library(
+    name = "uniset",
```

Comment:
In toolutil, I expect that most cc_library targets want to have a single .cpp file each.

In common and i18n, this approach should also work, but may not be necessary. My hunch is that making one cc_library target for each dependencies.txt group should be sufficiently granular.

E.g.
```
group: uniset_core
    unifilt.o unifunct.o
    uniset.o bmpset.o unisetspan.o
  deps
    patternprops
    icu_utility
    uvector
```

Reply:
Done.